pkg/internal/checkup/checkup.go

```diff
@@ -49,6 +49,7 @@ type Checkup struct {
 	client    kubeVirtVMIClient
 	namespace string
 	vmi       *kvcorev1.VirtualMachineInstance
+	results   status.Results
```

Comment:
Please consider squashing / moving this commit closer to the commit introducing the executor.

Reply:
it makes more sense to move it to just before the executor is introduced.
DONE